scripts/itineraries/ItinerarySaveButton.js

```diff
@@ -0,0 +1,70 @@
+import { saveItinerary } from "./ItineraryProvider.js";
+
+const contentTarget = document.querySelector(".saveItineraryButtonContainer")
+const eventHub = document.querySelector(".container")
+
+let isParkSelected = false
+let isEaterySelected = false
+let isOdditySelected = false
+let iteneraryData = {
```

Comment:
change to itineraryData

Reply:
Oops. Lots of mispellings in this file. Tried to catch them in an earlier commit, but I guess I fumbled the save and commit process.